docs/reference/oci.md

```diff
@@ -49,6 +49,14 @@ Its content must be a valid ELF file.
 There must be at most one layer with the wasm media type. If present, it must
 not be empty and it must be a valid wasm file.
 
-## Image labels
+## Image annotations
 
-TODO
+OCI images can have annotations at different levels:
+- index
+- manifest
+- config
+- layer
+
+Inspektor Gadget adds the following annotations at the manifest and config levels:
+- `io.artifacthub.package.readme-url`: defined by [ArtifactHUB](https://artifacthub.io/docs/topics/repositories/container-images/#image-metadata)
```

Comment:
Following the discussion on https://github.com/artifacthub/hub/issues/3665 (https://github.com/inspektor-gadget/inspektor-gadget/issues/2175):
New kinds of artifacts are encouraged to use the file [artifacthub-pkg.yml](https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml) file. So gadget authors would publish their gadgets on Artifact Hub by adding a this file in their git repository. Artifact Hub would not fetch the annotations from the OCI image.

So I will remove this artifacthub annotation for now.

Note that users can always add custom OCI annotations in the `annotations` field of `gadget.yaml`.